### PR TITLE
adds edit functionality, CSS WIP

### PR DESCRIPTION
**Pull request by kousagi1012**

### CODE REVIEW

File: public/stylesheets/style.css [hunk 1]
```diff
@@ -3,23 +3,40 @@ body {
   font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
 }
 
-.item {
+/*.item {
```
Comment (shakalee14) [inline public/stylesheets/style.css]: why are you keeping commented code?

File: public/stylesheets/style.css [hunk 2]
```diff
@@ -3,23 +3,40 @@ body {
...
+.description {
+  display:block;
+  position:relative;
+}
+
+/*.action {
+  float: right;
 }
 
-.dButt {
```
Comment (shakalee14) [inline public/stylesheets/style.css]: the death of dButt

File: routes/index.js [hunk 3]
```diff
@@ -19,15 +19,26 @@ router.post('/send', function(request, response, next) {
 router.post('/modify', function(request, response, next) {
   const {del} = request.body
```
Comment (shakalee14) [inline routes/index.js]: ``` const {del, complete, uncomplete, name, id } = request.body ```

Comment (shakalee14) [inline routes/index.js]: instead of what you have

File: views/index.pug [hunk 4]
```diff
@@ -8,10 +8,16 @@ block content
...
   for x in task
-    form(method="post" action="/modify" class="list")
-      div(id=x.id class="item")= x.task
-        span(class="dButt")
-          button(type="submit" name="complete" value=x.id) Complete
-          button(type="submit" onclick="return confirm('are you sure you want to delete this?')" name="del" value=x.id) X
+    span(class="item")
+      span(class="description" id=x.id class=x.complete?"completedItem":"item")= x.task
+      span(class="action")
+        if x.complete
```
Comment (shakalee14) [inline views/index.pug]: nice is this how you're doing the toggling?

File: public/stylesheets/style.css [hunk 5]
```diff
@@ -3,23 +3,40 @@ body {
...
+.list {
+  float: right;
+}*/
+
+.completedItem {
+  text-decoration: line-through;
+  color: red;
+
```
Comment (shakalee14) [inline public/stylesheets/style.css]: extra line here can be removed

File: public/stylesheets/style.css [hunk 6]
```diff
@@ -3,23 +3,40 @@ body {
...
+}
+
+/*.list {
```
Comment (shakalee14) [inline public/stylesheets/style.css]: why is this included?

File: views/index.pug (hunk 4 above)
Comment (kousagi1012) [inline views/index.pug]: The ternary is to set the class as either "completedItem" if the property complete in the database is true, otherwise set the class as "item" so we can target them with CSS; the toggling is being done with our query

File: public/stylesheets/style.css (hunk 2 above)
Comment (kousagi1012) [inline public/stylesheets/style.css]: a sad day

File: routes/index.js (hunk 3 above)
Comment (kousagi1012) [inline routes/index.js]: cool!

File: public/stylesheets/style.css (hunk 1 above)
Comment (sashajustice) [inline public/stylesheets/style.css]: Is it bad to do that? I'm takin notes so I remember stuff for next time I do this project
